includes/class-llms-block-templates.php

```diff
@@ -0,0 +1,564 @@
+<?php
+/**
+ * Handles the block templates
+ *
+ * @package LifterLMS/Classes
+ *
+ * @since [version]
+ * @version [version]
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * LLMS_Block_Templates class.
+ *
+ * @since [version]
+ */
+class LLMS_Block_Templates {
+
+	/**
+	 * Directory name of the block templates
+	 *
+	 * @var string
+	 */
+	const LLMS_BLOCK_TEMPLATES_DIRECTORY_NAME = 'block-templates';
+
+	/**
+	 * Block Template namespace.
+	 *
+	 * This is used to save templates to the DB which are stored against this value in the wp_terms table.
+	 *
+	 * @var string
+	 */
+	const LLMS_BLOCK_TEMPLATES_NAMESPACE = 'lifterlms/lifterlms';
```

Comment:
I used that because (looking at wc) I found that WP automatically turns `lifterlms/lifterlms` (that we use as the `theme` block template property) into LifterLMS (you know, plugin name etc.).
But I think we can, as you propose, go with your suggestion and use this that I called `namespace` to build the slug. Then introduce a new constant like:
`const LLMS_BLOCK_TEMPLATES_THEME = 'lifterlms/lifterlms;`
so everybody's happy and makes more sense.

Reply:
going to revert this because the first part of the slug and the "theme name" are somewhat tied, also, what will be saved in the DB (which might occupy more space) is the theme name, and that - for the reason above - I would like to keep it as `lifterlms/lifterlms`.